### DISCUSSION TITLE
Transaction : Success - Unspent

### DISCUSSION THREAD

**Discussion by vbabenko** [Chia-Network/chia-blockchain#10060] [Support]
Hi there,
I tried to transfer a small amount of Chia for testing from Exchange to the official Chia app (wallet, downloaded from here: https://www.chia.net/download/). 
And the status of the transaction is: "Success - Unspent".
And I do not see anything in my wallet (No pending transactions, no Balance, etc...)
Transaction details: https://xchscan.com/txns/0xf8fcbd2a81e30cdbe675c6f083b5055611c5b38caf6b9112c8a4e800e7aaf831

Transaction details on the Exchange:
```
Address
xch1xge85umxepfwdkktrgvuewy89svp4jfjlhdgvk49krczaz0yvzxquezy2m
TxID
f8fcbd2a81e30cdbe675c6f083b5055611c5b38caf6b9112c8a4e800e7aaf831
Chain type
XCH
```

Screen of wallet page: 
![image](https://user-images.githubusercontent.com/4083910/151938162-48e5176e-1a6c-410e-8534-1168b9a4621c.png)

Could you pls help?

**Comment by vbabenko** [comment]
Once I sync the wallet I see the transaction. I've spent a week to sync the wallet...

**Reply by BrandtH22** [reply]
Hey @artemlyndov , there are two ways to resync the wallet:
1. In the GUI settings \ advanced there is a resync option, this will close the chia client and once you reopen it will begin the resync process
2. You can stop chia,  delete (or rename) the wallet db here ~\.chia\mainnet\wallet\db\ , then start chia and the resyncing will start
NOTE - the wallet db stores offer files and cat/did names so those will be removed with the second option

**Comment by vlad-adsd** [comment]
Hi @vbabenko , how long did it take for the transaction status to change to Success - spent? How long did it take you to get the coins in your account? I just have the same problem.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
